### PR TITLE
refactor: lint shell for shellcheck

### PR DESCRIPTION
**Pull request by zaunist**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Refer to [shellcheck](https://github.com/koalaman/shellcheck) to standardize shell scripting.
example:
```
pwd=`pwd`
```
to:
```
pwd=$(pwd)
```

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by zaunist** [comment]
cc @starsz @nic-6443 @bzp2010 @iamayushdas

**Comment by bzp2010** [comment]
@zaunist I think we can leave the changes to `cli_test.sh` here, it will cause a conflict with another PR that actually refactors it.

**Review by iamayushdas — APPROVED** [review]

**Review by nic-6443 — APPROVED** [review]

**Review by starsz — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by iamayushdas — APPROVED** [review]

**Comment by zaunist** [comment]
cc @bzp2010

File: api/run.sh [hunk 1]
```diff
@@ -17,8 +17,8 @@
 #
 
 export ENV=local
-pwd=`pwd`
+pwd=$(pwd)
 
-cd ./output
+cd ./output || exit
```
Comment (nic-chen) [inline api/run.sh:22]: should give some message

Comment (zaunist) [reply api/run.sh:22]: If `cd` fails, the system will give the error message `No such file or directory exists`

Comment (zaunist) [reply api/run.sh:22]: And the script will exit after `cd` fails to execute, so there is no need to print some message

**Review by nic-chen — APPROVED** [review]

**Pull request merged**